### PR TITLE
Technopig/Balance&Bugs

### PR DESCRIPTION
**Pull request by Technopig1992**
General
Goblin Empire reward text updated.
Reduced cost of Navigation 250>75.
Reduced cost of Flight 100>75.
Increased the gold cost of Halaar and Shattrath 60>90.
Increased the gold cost of the Centaur Tents around Crossroads 0>30/40.
Excluded Structures from CP objectives.
Pillage Changes
- Salvage Cost Ratio increased 60>75%. (Total gold gained from building base cost)
Aeires Peak is now a T3 Capital.
Lightning Shield buffed. Mana cost 100>75, damage per second 20>25, duration 10>20. (effects Ysondre, Furbolg Shamen and Ashtongue Mystic.
Moved multiple creeps that would aggro unintentionally.
Removed some creeps in the following areas:
- STV
- Blackrock Mountain
- Grim Batol
- Lakeshire
- Darkshire

Stormwind
Safeguard ability nerfed, damage taken increased 20%>35%.
Clergyman Dispel now in line with other Dispels (reduced damage to summoned units and aoe)
Aura of Protection name changed to Protection Aura.
Icon changed for Protection Aura.
New buff for Proection Aura.
New model for Protection Aura.

Warsong
Blademaster Bladestorm duration increased 0.75>0.8 seconds.
Orgrimmar CP value increased 25g>30g.

Frostwolf
Voljin's Serpent Wards cast range increased 500/500/500/500 > 600/600/600/700.

Kul Tiras
Increased mana cost of Garrison ability 0>100.
Cluster Bomb cast range reduced 600>400.
Cluster Bomb cast time increased 0.00>0.50 seconds.
Cluster bombs stun duration decreased 0.25>0.2 seconds.
T2 now has mana 0/200.
T3 now has mana 300/300.
T2 & T3 now regen mana 0>2.25.

Fel Horde 
Fel Stalker Demon Gate # spawned reduced 2>1.
Unfocused Demon Gates are now free to upgrade 50g>0g.

Druids
Increased mana cost of Summon Treants ability (T2,T3) 0>100.
Removed one upgrade level from Natures Blessing.
Ancient Protector Defense upgrade bonus reduced 5>2.
Ancient Protector Attack Speed 1 increased 1.2>2.00.
Improved Ancient Protecter Defense upgrade bonus reduced 5>2.
Improved Ancient Protector Attack Speed 1 increased 1.2>2.00.

Bugs
Ghouls and Half Ghouls can no longer Harvest or Attack trees.
Fixed a bug where you could exploit high armor values when a unit was under the effects of both Aura of Protection and Devotion Aura.
Druid of the Talon (Storm Crow form) now benefits from attack and defence upgrades.
Control Point Resistance now correctly reduces damage taken by magic attacks on CP Towers to 100%.
Updated Shores of Feathermoon reward text.
Control Point Resistance tooltip
Corrected Tooltips:
- Make it Rain
- Bladestorm
- Cluster Bombs
- Control Point Resistance.
- Kirin Tor.
- Intercontinental Artillery.

### CODE REVIEW

**Comment by Technopig1992** [comment]
@SebastianTanoni Added a reduction for Navigation and Flight costs - but wanted to run it by you first

File: src/WarcraftLegacies.Source/Quests/Sentinels/QuestFeathermoon.cs [hunk 1]
```diff
@@ -37,8 +37,8 @@ public sealed class QuestFeathermoon : QuestData
       "Feathermoon Stronghold has been relieved and has joined the Sentinels in their war effort";
 
     /// <inheritdoc />
-    protected override string RewardDescription =>
-      "Control of all units in Feathermoon Stronghold and enable Tyrande to be trained at the Altar of Watchers";
+    protected override string RewardDescription => 
+      $"Upon completion, you will gain control of all units in Feathermoon Stronghold, including the ability to train {GetObjectName(UNIT_EBAL_GLAIVE_THROWER_SENTINELS)}s from the {GetObjectName(UNIT_E00L_WAR_ACADEMY_SENTINEL_BARRACKS)}. Additionally, you can now train Tyrande at the {GetObjectName(UNIT_E00R_ALTAR_OF_WATCHERS_SENTINEL_ALTAR)}";
```
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Quests/Sentinels/QuestFeathermoon.cs:41]:
```suggestion
      $"Gain control of all units in Feathermoon Stronghold, learn to train {GetObjectName(UNIT_EBAL_GLAIVE_THROWER_SENTINELS)}s from the {GetObjectName(UNIT_E00L_WAR_ACADEMY_SENTINEL_BARRACKS)}, and learn to train Tyrande from the {GetObjectName(UNIT_E00R_ALTAR_OF_WATCHERS_SENTINEL_ALTAR)}";
```
Keep it simple!

**Comment by YakaryBovine** [comment]
I merged `main` back into your branch and resolved merge conflicts by taking changes from `main`. You will want to make sure your changes to Make It Rain are still there, and that your creep camps are still where you want them.

**Comment by Technopig1992** [comment]
Awesome will do ty!

**Comment by Technopig1992** [comment]
@YakaryBovine does the tooltip for Control Point Resistance seem correct to you? feels off to me but couldn't think of something more fitting

**Comment by YakaryBovine** [comment]
Marking this as draft. As discussed on Discord just now, we don't want to make the Kalimdor centaur tents hostile all the time, because Warsong is about to become a choice faction. Let's revert those changes, then do them again as a seperate PR since it's complicated.

**Comment by Technopig1992** [comment]
image

image

image

**Review by YakaryBovine — APPROVED** [review]
Looks great. Thanks heaps for the Protection Aura screenshots, saved me from having to get home and test the map again!

**Pull request merged**